tutorials/transparent-proxy-and-filtering-on-k8s-with-initializers/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Transparent Proxy and Filtering on Kubernetes with Initializers
+description: Learn how to run a transparent proxy on Kubernetes to filter and intercept traffic out of your deployments using custom deployment initializers.
+author: danisla
+tags: Kubernetes mitmproxy proxy
+date_published: 2017-09-23
+---
+
+Dan Isla | Google Cloud Solution Architect | Google
+
+This is a follow-on tutorial to the [Transparent Proxy and Filtering on K8S](https://cloud.google.com/community/tutorials/transparent-proxy-and-filtering-on-k8s) tutorial. It shows how to simplify the application of a transparent proxy for existing deployments using a Deployment Initializer. Initializers are one of the [Dynamic Admission Control](https://kubernetes.io/docs/admin/extensible-admission-controllers/) features of Kubernetes, and are available as an alpha feature in Kubernetes 1.7.
```

Comment:
K8S &rarr; Kubernetes

Reply:
also need to match the URL with the deployed link of the other tutorial, @jmdobry do you know what that will be?

Reply:
#251 will end up at https://cloud.google.com/community/tutorials/transparent-proxy-and-filtering-on-k8s

#252 will end up at https://cloud.google.com/community/tutorials/transparent-proxy-and-filtering-on-k8s-with-initializers

Reply:
Great, thanks!

Reply:
I fixed the K8S -> Kubernetes